SCClassLibrary/JITLib/ProxySpace/ProxyInterfaces.sc

```diff
@@ -413,6 +413,9 @@ SynthDefControl : SynthControl {
 		bytes = control.bytes; // copy cached data
 	}
 
+	findSpecFor { |controlName|
+		^synthDef.findSpecFor(controlName)
+	}
 	specs {
 		^synthDef.specs
 	}
```

Comment:
You could even add one for `SynthControl`:

```supercollider
findSpecFor { |controlName|
var specs = this.synthDesc.specs;
^specs !? { specs[controlName] }
}
```

Then you can use the specs of `Synth`s played like `Ndef(\x, \default)`

Reply:
Should SynthDesc also have its own  findSpecFor method?
SynthDesc is used in many other places, so could be useful, no?

Reply:
yes, that seems reasonable.